src/components/Select/components/SelectControl/SelectControl.tsx

```diff
@@ -105,6 +111,18 @@ export const SelectControl = React.forwardRef<HTMLButtonElement, ControlProps>((
         clearValue();
     }, [clearValue]);
 
+    const renderCounterComponent = () => {
+        if (!hasCounter) {
+            return null;
+        }
+        const count = Number(value?.length) || 0;
+        const counterComponent = <SelectCounter count={count} size={size} disabled={disabled} />;
+        if (renderCounter) {
+            return renderCounter(counterComponent, {count, size, disabled});
+        }
+        return counterComponent;
```

Comment:
I think we can use ternary operator here, smth like
```tsx
const renderCounterComponent = () => {
  if (!hasCounter) {
    return null;
  }
  
  const count = Number(value?.length) || 0;
  
  return renderCounter
    ? renderCounter(counterComponent, {count, size, disabled})
    : <SelectCounter count={count} size={size} disabled={disabled} />;
};
```

Reply:
I made it by analogy with render control code.
Yes, with ternary operator will be better
[69f2a35](https://github.com/gravity-ui/uikit/pull/1368/commits/69f2a35782624557cbbd41c223c177c3f6ff0713)